logger/log_formatter.go

```diff
@@ -0,0 +1,65 @@
+package logger
+
```

Comment:
Apologies for the nits :-) mind naming this file `gcp_formatter.go` ?

Reply:
no problem!